graphs/lca.cpp

```diff
@@ -48,6 +52,8 @@ struct lca {
 	}
 
 	int distEdges(int x, int y) {
```

Comment:
lets change the name to match what it does

Reply:
also return a ll

Reply:
also can we keep a "distance in edges" function in addition to the new distance function?

We may need both